contracts/governance/ValidatorSetSig.sol

```diff
@@ -87,7 +89,9 @@ contract ValidatorSetSig is ReentrancyGuard {
         blockchainID = WARP_MESSENGER.getBlockchainID();
     }
 
-    function executeCall(uint32 messageIndex) external nonReentrant {
+    receive() external payable {}
+
+    function executeCall(uint32 messageIndex) external payable nonReentrant {
```

Comment:
For my own understanding, is `payable` here strictly necessary in order to specify a value in the below `call`? If this was not `payable`, would the behavior be that this function is callable by anyone (assuming the contract itself has the required funds, provided through `receive`)

Reply:
It is not strictly necessary, as long as the contract has enough funds the `call` inside `executeCall` will be able to send them along to the target. It was made payable as a convenience to give two different ways of sending native tokens to the target contract.